app/app/settings.py

```diff
@@ -516,6 +516,12 @@
 GITHUB_API_TOKEN = env('GITHUB_API_TOKEN', default='')  # TODO
 GITHUB_APP_NAME = env('GITHUB_APP_NAME', default='gitcoin-local')
 
+# Etherscan API
+ETHERSCAN_API_KEY = env('ETHERSCAN_API_KEY', default='')
+
+# Kudos revenue account
+KUDOS_REVENUE_ACCOUNT_ADDRESS = env('KUDOS_REVENUE_ACCOUNT_ADDRESS', default='0xdb282cee382244e05dd226c8809d2405b76fbdc9')
```

Comment:
@owocki is this the correct address (and is it the same on rinkeby, or is there even a rinkeby equivalent?)

Reply:
no rinkeby equiv; this is the right addr yes